cypress/integration/projectlistingpage.spec.js

```diff
@@ -0,0 +1,10 @@
+describe("Project Detail page", () => {
```

Comment:
Shouldn't this be describe(Project Listing page)... et cetera

Reply:
thanks, just updated